### PR TITLE
chore: Upgrade Netlify to Forma4

### PR DESCRIPTION
**Pull request by colomolo**

### CODE REVIEW

File: apps/netlify/src/index.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import ReactDOM from 'react-dom';
 
 import { init, locations } from '@contentful/app-sdk';
 
-import '@contentful/forma-36-react-components/dist/styles.css';
+// import '@contentful/f36-components/dist/styles.css';
```
Comment (Jwhiles) [inline apps/netlify/src/index.js:8]: ?

**Review by Jwhiles — COMMENTED** [review]
Changes look good - though theres a commented out styles import that I'm not a bit confused about

File: apps/netlify/src/index.js (hunk 1 above)
Comment (colomolo) [reply apps/netlify/src/index.js]: Thanks, removed that.

**Review by Jwhiles — COMMENTED** [review]
Looks like somethings gone a bit weird here - theres 134 commits, and a lot of conflicts

**Comment by kdamball** [comment]
As @Jwhiles mentioned, we might need to do some commit cherry-picking as the other branch has split netlify code into frontend/lambda

**Comment by colomolo** [comment]
Yep, that's I'm busy with right now.

**Comment by colomolo** [comment]
@Jwhiles @kdamball Fixed PR: https://github.com/contentful/apps/pull/892

**Pull request closed without merging**